sdk/communication/communication-rooms/CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
```

Comment:
Let's change this back to Unreleased. It should be the actual day when we release the JS SDK package.